code/modules/mob/living/simple_animal/hostile/terror_spiders/actions.dm

```diff
@@ -196,7 +196,7 @@
 					choices += L
 		for(var/obj/O in oview(1,src))
 			if(Adjacent(O) && !O.anchored)
-				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling))
+				if(!istype(O, /obj/structure/spider/terrorweb) && !istype(O, /obj/structure/spider/cocoon) && !istype(O, /obj/structure/spider/spiderling/terror_spiderling) && !istype(O, /obj/effect))
```

Comment:
Are you sure, Crazy?

/obj/effect includes things like bluespace portals, beams, forcefields, immovable rods, meteors, blood runes, sparks, steam, smoke, etc.

I don't see why any of these things should be wrappable.